### DISCUSSION TITLE
determinate progress-indicator

### DISCUSSION THREAD

**Discussion by movrack** [ing-bank/lion#1043] [General]
Hello,

I would appreciate to have a progress bar with indicator status.
My need is for loading of files, basically our tool generate a zip server side, but I have following steps:
verifying data, creation of file, backing up somewhere, zipping it and sending it

I could do custom made spinner with current action below, our use step indicator but I don't think it fit the concern.

Apparently there is already some work on it (initial mr ?)
https://github.com/ing-bank/lion/pull/857#discussion_r465574254

**Comment by MathieuPuech** [comment]
With current progress indicator, you can create a progress bar indicator, it’s almost only styling. I did it for bar, circle and skeleton.

**Reply by daKmoR** [reply]
@MathieuPuech I think it would be awesome to add those as possible variations.. same as we did for combobox
